superbench/runner/ansible.py

```diff
@@ -59,27 +61,34 @@ def __init__(self, config):
                 self._config['cmdline'] += ' --ask-pass --ask-become-pass'
         logger.info(self._config)
 
-    def run(self, ansible_config, sudo=False):    # pragma: no cover
+    def run(self, ansible_config, sudo=False, stdout=False):    # pragma: no cover
```

Comment:
Is it another new feature?

Reply:
Has been added to description as a minor revision, do I need to raise another PR for this?

Reply:
Yes, also what's the goal for this?